opencti-platform/opencti-front/src/private/components/events/stix_sighting_relationships/EntityStixSightingRelationships.tsx

```diff
@@ -186,39 +250,15 @@ const EntityStixSightingRelationships: FunctionComponent<EntityStixSightingRelat
   return (
     <div className={classes.container}>
       {renderLines()}
-      <Security needs={[KNOWLEDGE_KNUPDATE]}>
-        {isTo ? (
-          <StixSightingRelationshipCreationFromEntity
-            entityId={entityId}
-            isTo={true}
-            stixCoreObjectTypes={[
-              'Threat-Actor',
-              'Intrusion-Set',
-              'Campaign',
-              'Malware',
-              'Tool',
-              'Vulnerability',
-              'Indicator',
-            ]}
-            targetStixCyberObservableTypes={['Stix-Cyber-Observable']}
-            paddingRight={noPadding ? null : 220}
-            paginationOptions={finalPaginationOptions}
-            variant={undefined}
-            onCreate={undefined}
-          />
-        ) : (
-          <StixSightingRelationshipCreationFromEntity
-            entityId={entityId}
-            isTo={false}
-            stixCoreObjectTypes={stixCoreObjectTypes}
-            targetStixCyberObservableTypes={undefined}
-            paddingRight={noPadding ? null : 220}
-            paginationOptions={finalPaginationOptions}
-            variant={undefined}
-            onCreate={undefined}
-          />
-        )}
-      </Security>
+      {!FABReplaced
+        && <SightingCreationComponent
+          isTo={isTo}
+          entityId={entityId}
+          noPadding={noPadding}
+          paginationOptions={finalPaginationOptions}
+          stixCoreObjectTypes={stixCoreObjectTypes}
+           />
+      }
```

Comment:
We prefer this formatting for consistency (but I see our linter is not properly configured for JSX parts).

```suggestion
      {!FABReplaced && (
        <SightingCreationComponent
          isTo={isTo}
          entityId={entityId}
          noPadding={noPadding}
          paginationOptions={finalPaginationOptions}
          stixCoreObjectTypes={stixCoreObjectTypes}
        />
      )}
```